build.gradle

```diff
@@ -105,18 +106,33 @@ configure(javaProjects) {
             dependency("org.opensaml:opensaml:2.6.4")
             dependency("org.python:jython-standalone:2.7.1")
             dependency("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
+            dependencySet(group: "com.google.protobuf", version: "3.4.0") {
+              entry "protobuf-java"
+              entry "protoc"
+            }
+            dependencySet(group: "io.grpc", version: "1.6.1") {
+              entry "grpc-core"
+              entry "grpc-netty"
+              entry "grpc-stub"
+              entry "grpc-protobuf"
+              entry "protoc-gen-grpc-java"
+              entry "grpc-testing"
+            }
         }
     }
 
     ext {
         generatedSourcesDir = new File("${projectDir}/src/generated")
         generatedSourcesJavaDir = new File(generatedSourcesDir, "/java")
+        // Necessary due to limitations of proto plugin (${source-set-name}/${variant} is always appended)
```

Comment:
Why does this have to be in `src/generated/main/java`?  why not just `src/generated/java` like the other one? I might be missing the point of this comment. Seems to conflict with what you have down in the plugin configuration below